Features/Context/SubContext/CommonActions.php

```diff
@@ -0,0 +1,363 @@
+<?php
+
+/**
+ * File containing the Common Funtios context class for PlatformUI.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\PlatformUIBundle\Features\Context\SubContext;
+
+use EzSystems\PlatformUIBundle\Features\Helper\JavaScript as JsHelper;
+
+trait CommonActions
+{
+    /**
+     * Waits for Javascript to finnish by running a empty Javascript
+     * (In Sahi it's possible to have the same result by running an empty javascript only)
+     */
+    protected function waitForLastJs()
+    {
+        //Needs to be here
+        $this->execJavascript( '' );
+
+        $jsCode = "return BDD.isSomethingLoading();";
+
+        while ( $this->evalJavascript( $jsCode ) )
+        {
+            usleep( 100 * 1000 ); // 100ms
+        }
+    }
+
+    /**
+     * Waits for Javascript to finnish by running a empty Javascript
+     * (In Sahi it's possible to have the same result by running an empty javascript only)
+     */
+    protected function activateJsErrorHandler()
+    {
+        //Needs to be here
+       // $this->execJavascript( '' );
+        $jsCode = "return BDD.errorHandlerActivate();";
+        $this->execJavascript( $jsCode );
+    }
+
+    /**
+     * Evaluate javascript code and return result.
+     *
+     * @param  string  $jsCode
+     * @return mixed
+     */
+    protected function evalJavascript( $jsCode, $addJsHelper = true )
+    {
+        if ( $addJsHelper )
+        {
+            $jsCode = JsHelper::getHelperJs() . $jsCode;
+        }
+        $jsCode = $this->driver->wrappedFunction( $jsCode );
+
+        return $this->getSession()->evaluateScript( $jsCode );
+    }
+
+    /**
+     * Execute javascript code and return result.
+     *
+     * @param  string  $jsCode
+     * @return mixed
+     */
+    protected function execJavascript( $jsCode, $addJsHelper = true )
+    {
+        if ( $addJsHelper )
+        {
+            $jsCode = JsHelper::getHelperJs() . $jsCode;
+        }
+
+        $jsCode = $this->driver->wrappedFunction( $jsCode );
+
+        return $this->getSession()->executeScript( $jsCode );
+    }
+
+    /**
+     * Clicks on the PlatformUI logo
+     *
+     * @param string $text
+     */
+    protected function clickLogo()
+    {
+        $selector = 'img[alt="Logo eZ"]';
```

Comment:
yes, you are right I didn't spend much time on the logo sentence because it was not a priority at the time but I will fix it.